SCClassLibrary/Common/Control/Node.sc

```diff
@@ -146,14 +146,16 @@ Node {
 			var cmd, argnodeID, parent, prev, next, isGroup, head, tail;
 			# cmd, argnodeID, parent, prev, next, isGroup, head, tail = msg;
 			// assuming its me ... if(nodeID == argnodeID)
-			Post << if(isGroup == 1, "Group:" , "Synth:") << nodeID << Char.nl
-			<< "parent  : " << parent << Char.nl
-			<< "prev : " << prev << Char.nl
-			<< "next :" << next << Char.nl;
-			if(isGroup==1, {
-				Post << "head :" << head << Char.nl
-				<< "tail :" << tail << Char.nl << Char.nl;
-			});
+			Post
+				<< if(isGroup == 1, "Group:\t" , "Synth:\t") << nodeID << Char.nl
+				<< "parent:\t" << parent << Char.nl
+				<< "prev:\t" << prev << Char.nl
+				<< "next:\t" << next << Char.nl;
+			if(isGroup == 1) {
+				Post
+				<< "head:\t" << head << Char.nl
+				<< "tail:\t" << tail << Char.nl << Char.nl;
```

Comment:
these 2 lines should also be indented by 1 more tab

Reply:
I thought it was nice that you can just read them downwards.

Reply:
but then the indentation style is inconsistent, i think it is less clear this way. when code is more predictably formatted, it takes less effort to read. the rule you're establishing here does not have much precedent and is also IMO difficult to explain if you wanted it to be followed elsewhere.